src/sentry/utils/json.py

```diff
@@ -63,6 +63,11 @@ def better_default_encoder(o):
 )
 
 
+def dump(value, fp, **kwargs):
```

Comment:
We do this on all of the other ones to be API compatible, but we are enforcing our own params.